### ISSUE TITLE
Implement multi-output for non-multioutput-models

### ISSUE THREAD

**Issue by mastoffel** [alan-turing-institute/autoemulate#63]
Some models that would be good to implement (Support Vector Machines, Gradient Boosting) don't natively support multi-output. We can, however, wrap them in a `MultiOutputRegressor` (see [here](https://scikit-learn.org/stable/modules/generated/sklearn.multioutput.MultiOutputRegressor.html) which essentially fits one model per target (not leveraging correlation). The problem is that parameters will then have different names starting with `estimator__` so that we need to account for this during hyperparameter search.
